handler_test.go

```diff
@@ -12,6 +13,20 @@ import (
 	"github.com/nats-io/graft/pb"
 )
 
+type logPositionHandler struct {
+	logIndex uint32
```

Comment:
Should we just use `currentSequence` instead of `logIndex`? Also, should this be at least a `uint64`?